db/migrations/20230427114000_connect_read_pending_users_capabilities.json

```diff
@@ -1,11 +1,11 @@
 {
   "up": [
-    "REPLACE INTO group_capabilities (group_id, capabilities_id) VALUES",
-    "(1, (SELECT id FROM capabilities WHERE name = 'read_pending_users' AND is_system_supplied = true)),",
-    "(2, (SELECT id FROM capabilities WHERE name = 'read_pending_users' AND is_system_supplied = true));"
+    "INSERT INTO group_capabilities (group_id, capabilities_id) VALUES",
```

Comment:
I think this file has been fixed already because `INSERT` is now converted to `REPLACE`. Please double check if this change is necessary.